### DISCUSSION TITLE
Order of the next coupled simulations

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#154] [TF5: Coupled SE/MOM6]
This is a follow up on the CAM7 meeting discussion. 
At CAM7 meeting, we presented the results of the latest coupled simulations: 009, 016, 017, 018b, 019b and 020. 
The slides can be found [here](https://docs.google.com/presentation/d/1t-6N-5OTjpRRpVVcDt_SyhJ6Ne2JM1fugX0yaEupUR0/edit?usp=sharing). If you don't have permissions, please shoot Cecile an email. 

During the meeting, we discussed the order of the simulations and we decided to have a space to continue the discussion. 

Please pots your suggestions/comments here. 

Thanks

----
![Screen Shot 2022-08-05 at 12 46 49 PM](https://user-images.githubusercontent.com/9723220/183141962-f5a7b390-be91-4978-8741-c56713db9faf.png)
----

**Comment by JulioTBacmeister** [comment]
Runs 020 (+ cam_dev, https://github.com/NCAR/amwg_dev/issues/153 ) and 021 (+ cam_dev +no_leak_topo, https://github.com/NCAR/amwg_dev/issues/155 ) appear to be staying in the "pack" of acceptable runs.  See Figure in #155.  If sufficient Lab Sea/N Atlantic convection is present in 020 and 021 we will have two atmosphere mods left to test 

1) L32==> L58 
2) B1850==>BWsc1850

I suggest we start two runs at the same time 

022 = 021+BWsc1850
023 = 021+Bwsc1850+L58

This is in the spirit of checking off all the mods we expect to be harmless, before introducing what IMHO is the likeliest suspect - L58.
